### PR TITLE
P3 view alerts

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds documentation to explain to users how to view alerts.

Reviewers - please see comments for where I need more information.

### CODE REVIEW

File: docs/photoniq/p3/view-p3-alerts.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Alert Fields
+
+The following fields list information about the alert:
+
+- **URL** - The URL that the warning or error applies to.
+- **Status** - Current status. Possible statuses are: Failed, XXX, or XXX.
+- **Timestamp** - The time when the warning or error was logged.
+- **Policy Name** - The name of the policy that the warning or error is associated with.
+- **Action** - Possible actions for you to take. You can click **ACK** to acknowledge the warning or error, or ...
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-alerts.md]: What actions are possible, what does it mean if the user performs them, such as clicking ACK?

File: docs/photoniq/p3/view-p3-alerts.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+3. (Optional) In the Alerts table, click **ACK** to acknowledge the alert.
+
+![View P3 Alerts](/img/photoniq/p3/p3-view-alerts.png)
+
+## Alert Fields
+
+The following fields list information about the alert:
+
+- **URL** - The URL that the warning or error applies to.
+- **Status** - Current status. Possible statuses are: Failed, XXX, or XXX.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-alerts.md]: What are the possible statuses and what do they mean?

File: docs/photoniq/p3/view-p3-alerts.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can track P3 metrics on the dashboard.
+
+## View Alerts
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **Alerts & Audits**.
+3. (Optional) In the Alerts table, click **ACK** to acknowledge the alert.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-alerts.md:16]: What happens after they click ACK? Are there other actions they can take?

File: docs/photoniq/p3/view-p3-alerts.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 60
+title: View P3 Alerts
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can track P3 alerts on the dashboard.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-alerts.md]: What are we alerting on? We should tell the user what exactly is being tracked and might appear in this dashboard. Also, can they subscribe to the alerts somehow? I didn't see a corresponding endpoint in the API.

Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-alerts.md]: There will be an api which they can use to have it . We are alerting the user for now when the validation fails at in the offline optimizer phase

File: docs/photoniq/p3/view-p3-alerts.md (hunk 3 above)
Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-alerts.md:16]: When they clicj ack for now it means they have seen the message and are aware of it . No actions will be taken for now with respect to their ack. MM team will keep a track of those alerts and will try to resolve why they are happening

File: docs/photoniq/p3/view-p3-alerts.md (hunk 2 above)
Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-alerts.md]: Failed only for now

File: docs/photoniq/p3/view-p3-alerts.md (hunk 1 above)
Comment (dwivedianurag) [reply docs/photoniq/p3/view-p3-alerts.md]: ACK is the only action possible rn

**Review by dwivedianurag — APPROVED** [review]
ok

**Pull request merged**